Add Chopin Mandarin tutorial and update beta user info (#46)
apiobuild/apiobuild.github.io: content/sign-up/for-businesses.md

buttonLink: 'forms/business'
---

> Join us on the journey to reinvent how custom solutions are built. We are currently looking for beta users to test and expand product features of Chopin, a static online store generator. If you're bringing your business online but don't know how, get in touch with us!

Reviewer: ```suggestion
Join us on the journey to reinvent how custom solutions are built. We are currently looking for beta users to help us expand product features of Chopin - our static online store generator. If you're bringing your business online but don't know how, get in touch with us!
```
